### PR TITLE
added resource page and privacy policy page pre-population migration

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
- [x] Migrations
  - [x] Migration is backwards-compatible with current production code
- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Ticket #5025 

#### What's this PR do?
Adds privacy policy page.

#### How should this be manually tested?

Run migrations and visit /privacy-policy/
ensure page is visible and correct styles have been applied 

#### Screenshots (if appropriate)
Desktop Screen shots:

Screenshot 2021-09-03 at 10 17 08 AM

Mobile screent shots

Screenshot 2021-09-03 at 10 17 34 AM

Screenshot 2021-09-03 at 10 18 05 AM

CMS Page.

Screenshot 2021-09-07 at 10 24 59 AM

Screenshot 2021-09-07 at 10 25 15 AM

Screenshot 2021-09-07 at 10 25 27 AM

Screenshot 2021-09-07 at 10 25 45 AM

Screenshot 2021-09-07 at 10 25 54 AM

Footer on page.

Screenshot 2021-09-07 at 11 28 29 AM

### CODE REVIEW

**Comment by Wassaf-Shahzad** [comment]
(for prosperity)
I ran into an issue in micromasters, If I create a migration file for the resource page and restart the docker container it fails during applying migrations (0004_create_homepage) .However if I delete the migration file for resource page,The container restarts successfully.
If I create and apply migrations from a running container they succeed as well.
The error the docker container gives is a Null constraint failure on field locale_id while creating the home_page.
After a bit RnD I did work out a solution which was adding the following code to migration file (0004_create_homepage)
run_before = [
        ('wagtailcore', '0053_locale_model'),
    ]
in reference to https://stackoverflow.com/questions/65369636/wagtail-locale-id-cannot-be-null (edited)

File: static/scss/resource.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    &:after {
+      font-size: 24px;
+      font-family: Material Icons;
+      content: "arrow_circle_up";
+      margin-left: 5px;
+      vertical-align: middle;
+    }
+  }
+}
+
```
[the remark this answers is not in the document]
Comment (Wassaf-Shahzad) [reply static/scss/resource.scss:88]: I am sorry @pdpinch, I was unable to identify any css that we could inherit as most  of our styles in micromasters are scoped (in my knowledge).

**Review by pdpinch — COMMENTED** [review]
screenshots look good to me.

**Comment by briangrossman** [comment]
@Wassaf-Shahzad 
It looks like you're making good progress on this. It's looking very good. 

A couple notes: 
- Can you include the page header in the screenshot so we can see the page in context? 
- The link to the mobile design isn't working. Can you repost the mobile version? 
- Can you change the title to `Privacy Policy` (remove the hyphen `-`)
- Please make sure the red/cardinal color around `Privacy Policy` is the same color as the header background (I think it's slightly different right now)

**Comment by Wassaf-Shahzad** [comment]
> @Wassaf-Shahzad
> It looks like you're making good progress on this. It's looking very good.
> 
> A couple notes:
> 
> * Can you include the page header in the screenshot so we can see the page in context?
> * The link to the mobile design isn't working. Can you repost the mobile version?
> * Can you change the title to `Privacy Policy` (remove the hyphen `-`)
> * Please make sure the red/cardinal color around `Privacy Policy` is the same color as the header background (I think it's slightly different right now)

@briangrossman 

>* I have included the screenshot of the whole page.
>* Sorry for the that, I  have added mobile screenshots as well.
>* Updated title swell.
>* Changed the color around `Privacy Policy` and can be viewed in the new screenshot.

Thank you

**Comment by briangrossman** [comment]
@Wassaf-Shahzad 
These designs look good to me. 👍 
Nice job. 

Can you also provide a screenshot of the CMS page for administering this page as well as the updated footer?

**Comment by Wassaf-Shahzad** [comment]
> @Wassaf-Shahzad
> These designs look good to me. 👍
> Nice job.
> 
> Can you also provide a screenshot of the CMS page for administering this page as well as the updated footer?

Added requested screenshots.

**Comment by briangrossman** [comment]
> > @Wassaf-Shahzad
> > These designs look good to me. 👍
> > Nice job.
> > Can you also provide a screenshot of the CMS page for administering this page as well as the updated footer?
> 
> Added requested screenshots.

Thanks. Looks good to me. Thanks. 👍

**Review by annagav — CHANGES_REQUESTED** [review]
The sign in button doesn't seem to work on the Privacy Policy page.

**Comment by Wassaf-Shahzad** [comment]
> The sign in button doesn't seem to work on the Privacy Policy page.

Sorry about that, I have updated page as needed.

**Review by annagav — APPROVED** [review]
👍

**Pull request merged**